articles/libraries/standard/applications.md

```diff
@@ -182,29 +182,29 @@ Having seen how the quantum Fourier transform and phase estimation work (see [Qu
 Given two integers $a$ and $N$, where $a<N$, the goal of period finding, also called order finding, is to find the _order_ $r$ of $a$ modulo $N$, where $r$ is defined to be the least positive integer such that $a^r \equiv 1 \text{ mod } N$.  
 
 To find the order using a quantum computer, we can use the phase estimation algorithm applied to the following unitary operator $U_a$:
-$$ U_a|x\rangle \equiv |(ax)\text{ mod }N\rangle .$$
+$$ U_a\ket{x} \equiv \ket{(ax)\text{ mod }N} .$$
```

Comment:
One more catch, sorry, `\text{mod}` doesn't space the modulus operator correctly; I think that `\bmod` is preferred, but I haven't checked if we support that yet, sorry. In general, text commands in LaTeX should only be used for text, not operators and gates. Also happy to set that aside for future work, fixing incorrect use of `\text` definitely isn't the highest of priorities...